src/View/Helper/SchemaHelper.php

```diff
@@ -171,6 +171,14 @@ public function controlOptions($name, $value, $schema) : array
                 'checked' => $value,
             ];
         } elseif ($type === 'enum') {
+            if (!empty($schema['oneOf'])) {
+                foreach ($schema['oneOf'] as $one) {
+                    if (!empty($one['enum'])) {
+                        $schema['enum'] = ['-1' => ''] + $one['enum'];
```

Comment:
are we sure about this?

Reply:
`oneOf` for an enum... I guess it means that there are only 2 values, the first is `null`, the second is the list of enum values.
I couldn't say if there are other cases.

Reply:
Sorry, I was referring to `['-1' => '']` to set `null` enum item